Integration get raw history #443

luowenking1 opened the pull request.

meiliang86 commented on src/main/java/com/uber/cadence/internal/common/InternalUtils.java:
```diff
@@ -18,19 +18,18 @@
-import com.uber.cadence.SearchAttributes;
-import com.uber.cadence.TaskList;
-import com.uber.cadence.TaskListKind;
+import com.uber.cadence.*;
```
Please do not collapse the imports.

meiliang86 replied:
How do you test the new logic?

meiliang86 commented:
How do you test the new logic?

luowenking1 commented:
> How do you test the new logic?

Earlier the server change will always return the raw data, so I added the tests to mock the raw results and they are working fine. I removed them when we decides the change because of compatibility of server and client. However, I just revised the existing one for raw data deserialization testing which is working fine. For integration testing,  now the returned values from server depends on the value of a new flag SendRawHistoryData on server side. Seems we need to set up some different dockers for server and use them for testing, but not sure how to proceed with that yet. BTW, the pr on server was merged yesterday.
